builds/Makefile

```diff
@@ -1792,71 +1792,39 @@ $(BUILD_DESTDIR)/targettv/tmp-lib/Mono: $(BUILD_DESTDIR)/targettv/lib/libmonosge
 # bit version targeting arm64.
+# These are compiled using the mono sdk makefiles
 #
 
+SDK_CONFIG=$(MONO_PATH)/sdks/Make.config
+
+$(MONO_PATH)/sdks/Make.config:
+	echo "DISABLE_ANDROID=1" > $@
+
 #
-# we need to compile the cross build against llvm so we need to configure/build that first
+# Add explicit targets for building llvm before building the cross compilers, to
+# avoid building/downloading it in parallel by the nested make invocations.
 #
 ifdef INCLUDE_DEVICE
-cross:: build-llvm
-cross64:: build-llvm64
-
-setup:: setup-llvm setup-llvm64
-clean-local:: clean-llvm clean-llvm64
-
-setup-llvm: .stamp-configure-llvm
-setup-llvm64: .stamp-configure-llvm64
+clean-local:: clean-llvm
 endif
 
-LLVM_BASE_CONFIGURE_FLAGS=--enable-libcpp --enable-optimized --enable-assertions=no --disable-jit --disable-docs --disable-doxygen
-LLVM_CXXFLAGS=$(CCACHE_CXXFLAGS)
-LLVM_BASE_CONFIGURE_ENVIRONMENT=$(COMMON_ACVARS)
-
-# This is used just for linking with the 32 bit cross compiler
-# FIXME: Avoid building the libs not needed by mono
-.stamp-configure-llvm: $(LLVM_PATH)/configure
-	@mkdir -p llvm/
-	@echo "Configuring llvm"
-	@cd llvm && \
-	$(LLVM_BASE_CONFIGURE_ENVIRONMENT) \
-	$(abspath $(LLVM_PATH)/configure) --prefix=$$PWD/usr --build=i386-apple-darwin10.7  --enable-targets="arm" $(LLVM_BASE_CONFIGURE_FLAGS) --cache-file=../llvm.config.cache CC="$(CCACHE)clang" CXX="$(CCACHE)clang++" CXXFLAGS="$(LLVM_CXXFLAGS)" > $@.log 2>&1 || (echo "Configuring llvm failed:" && cat $@.log | sed "s/^/    /" && exit 1)
-	$(Q) @touch $@
-	@echo Configured llvm
-
-.stamp-configure-llvm64: $(LLVM_PATH)/configure
-	@mkdir -p llvm64/
-	@echo "Configuring llvm64"
-	@cd llvm64 && \
-	$(LLVM_BASE_CONFIGURE_ENVIRONMENT) \
-	$(abspath $(LLVM_PATH)/configure) --prefix=$$PWD/usr --enable-targets="arm arm64" $(LLVM_BASE_CONFIGURE_FLAGS) --cache-file=../llvm64.config.cache CC="$(CCACHE)clang" CXX="$(CCACHE)clang++" CXXFLAGS="-Qunused-arguments" > $@.log 2>&1 || (echo "Configuring llvm64 failed:" && cat $@.log | sed "s/^/    /" && exit 1)
-	$(Q) @touch $@
-	@echo "Configured llvm64"
-
-build-llvm: .stamp-build-llvm
-build-llvm64: .stamp-build-llvm64
-
-.stamp-build-llvm: .stamp-configure-llvm $(LLVM_DEPENDENCIES)
-	$(MAKE) -C llvm all install
-	$(Q) touch $@
+build-llvm32: .stamp-build-llvm
+build-llvm64: .stamp-build-llvm
 
-.stamp-build-llvm64: .stamp-configure-llvm64 $(LLVM_DEPENDENCIES)
-	$(MAKE) -C llvm64 all install
+.stamp-build-llvm: $(SDK_CONFIG)
+	$(MAKE) -C $(MONO_PATH)/sdks/builds build-ios-llvm
 	$(Q) touch $@
 
-really-build-llvm64:
-	$(MAKE) -C llvm64 all install
-	$(Q) touch .stamp-configure-llvm64
-
-clean-llvm:
-	-rm -rf llvm .stamp-configure-llvm .stamp-build-llvm llvm.config.cache
-clean-llvm64:
-	-rm -rf llvm64 .stamp-configure-llvm64 .stamp-build-llvm64 llvm64.config.cache
+clean-llvm: $(SDK_CONFIG)
+	$(MAKE) -C $(MONO_PATH)/sdks/builds clean-ios-llvm
+	$(RM) .stamp-*-llvm*
```

Comment:
`RM` isn't defined, so this would try to execute `-stamp-*-llvm*`.

`$(Q) rm -f .stamp-*-llvm*` should work fine though.

Reply:
RM is predefined make variable, its set to rm -rf.